mysql-delta-plugins/pom.xml

```diff
@@ -56,6 +56,10 @@
       <version>8.0.28</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Why do we need this ?

Reply:
Change Done, removed and added in dependencies tag of parent pom.